pkg/sqltransaction/sqltransaction.go

```diff
@@ -0,0 +1,39 @@
+// Package sqltransaction helps executing a function inside an SQL transaction.
+package sqltransaction
+
+import (
+	"context"
+	"database/sql"
+	"errors"
+	"fmt"
+
+	"github.com/nexmoinc/gosrvlib/pkg/logging"
+	"go.uber.org/zap"
```

Comment:
Are we using 2 logging libs at the same time?

Reply:
zap is used for the `zap.Error`. `logging` is mostly for context/initialisation

Reply:
No.